lib/config/presets/gitlab/index.ts

```diff
@@ -32,21 +32,21 @@ export async function fetchJSONFile(
   fileName: string,
   endpoint: string
 ): Promise<Preset> {
+  const urlEncodedRepo = encodeURIComponent(repo);
+  const urlEncodedPkgName = encodeURIComponent(fileName);
+  const defaultBranchName = await getDefaultBranchName(
+    urlEncodedRepo,
+    endpoint
+  );
```

Comment:
This can fail and should be moved back to try catch

Reply:
Thank you @viceice for the quick reply. I would open an issue, but I have already a look and I did not find any problem. 
In any case, what do you think about the improvement of logging?